### ISSUE TITLE
Underlined toggle buttons with per-instance properties

### ISSUE THREAD

**Issue by Chrriis** [JFormDesigner/FlatLaf#45]
Using toggle buttons that look like tabs is great for the L&F harmony of the application. This allows to create task bar buttons for example.

Now, depending on the context, the buttons will be adjusted in different ways (important bar may have bigger buttons, bold font, etc.). The problem is that the size of the underline and its related attributes are global. If I am not mistaken, there are no client properties to have a per-instance definition.

**Comment by DevCharly** [comment]
This is now implemented in master branch.

Use:
~~~java
toggleButton.putClientProperty( FlatClientProperties.TAB_BUTTON_SELECTED_BACKGROUND, Color.green );
toggleButton.putClientProperty( FlatClientProperties.TAB_BUTTON_UNDERLINE_COLOR, Color.red );
toggleButton.putClientProperty( FlatClientProperties.TAB_BUTTON_UNDERLINE_HEIGHT, 5 );
~~~

**Comment by DevCharly** [comment]
Forgot to mention that I've renamed toggle button type "underline" to "tab". Value of client property `JButton.buttonType` is now `tab`. All UI defaults renamed from `ToggleButton.underline.*` to `ToggleButton.tab.*`.

**Comment by Chrriis** [comment]
Good! I wanted to suggest this change from "underline" to "tab" 🙂

**Comment by DevCharly** [comment]
implemented in 0.26
